Doc/library/datetime.rst

```diff
@@ -2390,7 +2390,6 @@ requires, and these work on all platforms with a standard C implementation.
 |           | ``±HH:MM[:SS[.ffffff]]``       | -04:00, +10:30,        |       |
 |           | (empty string if the object is | +06:34:15,             |       |
 |           | naive).                        | -03:07:12.345216       |       |
-|           | .. versionadded:: 3.12         |                        |       |
```

Comment:
- It would still be useful to add an indicator of when this was added. If you look below you'll see `versionadded:: 3.6` mentioning that `%G` and a few others were added. I'd add another `versionadded:: 3.12` mentioning `%:z`.
- This table is preceded by a heading claiming it contains all the format codes required by the 1989 C stancard. I presume `%:z` is not in that standard. So it should probably be moved to the following table, which lists additional directives not in C89.
- Below all that there are a bunch of notes. Note (6) describes `%z` and `%Z`. You might want to add `%:z` there.

Reply:
done, see recent commits.